mevislab.github.io/content/tutorials/visualization.md

```diff
@@ -14,13 +14,17 @@ menu:
 ---
 # Visualization in MeVisLab {#TutorialVisualization}
 ## Introduction
-MeVisLab contains a whole toolkit to visualize data and images. 2D, 3D and 4D rendering is possible as well as the interaction with images and data objects.
 
-In this chapter, we focus on 2D and 3D visualization and simple image interactions. Not only pixel- and voxel-based data can be visualized, but also scene objects and 3D scenes (see [here](/tutorials/openinventor#TutorialOpenInventorModules)).
+Images and data objects can be rendered in 2D, 3D and 4D and interacted with in several ways using a set of tools available through MeVisLab. 
```

Comment:
I would remove 4D here - you can't really render in 4D...

Reply:
I just kept it in as it was there before - will be removed.